packages/devextreme/js/__internal/ui/m_file_uploader.ts

```diff
@@ -861,19 +864,30 @@ class FileUploader extends Editor<FileUploaderProperties> {
     this._isCustomClickEvent = false;
   }
 
-  _attachSelectFileDialogHandler(target) {
+  _selectFileDialogKeyUpHandler(e): void {
```

Comment:
Btw maybe we don't need a separate method for such a simple task - just run this `if` in arrow function when subscribe?

Reply:
https://github.com/DevExpress/DevExtreme/pull/28947#discussion_r1954548220